torch/csrc/jit/tensorexpr/expr.h

```diff
@@ -10,6 +10,8 @@
 #include <torch/csrc/jit/tensorexpr/mem_arena.h>
 #include <torch/csrc/jit/tensorexpr/types.h>
 
+#include <ATen/core/functional.h>
```

Comment:
Is it only for `fmap`? You could use `ExprVectorToExprHandleVector` if that doesn't look too ugly :)

Reply:
Yeah just for fmap.  I have this allergy to super long function names :-p